samples/cpp/benchmark_app/README.md

```diff
@@ -145,7 +145,39 @@ Options:
+    -report_type "<type>"       Optional. Enable collecting statistics report. "no_counters" report contains configuration options specified, resulting FPS and latency.
+                                "average_counters" report extends "no_counters" report and additionally includes average PM counters values for each layer from the network.
+                                "detailed_counters" report extends "average_counters" report and additionally includes per-layer PM counters
+                                and latency for each executed infer request.
+    -report_folder              Optional. Path to a folder where statistics report is stored.
+    -exec_graph_path            Optional. Path to a file where to store executable graph information serialized.
+    -pc                         Optional. Report performance counters.
+    -dump_config                Optional. Path to JSON file to dump IE parameters, which were set by application.
+    -load_config                Optional. Path to JSON file to load custom IE parameters. Please note, command line parameters have higher priority than parameters from configuration file.
+                                Example 1: a simple JSON file for HW device with primary properties.
+                                          {
+                                             "CPU": {"NUM_STREAMS": "3", "PERF_COUNT": "NO"}
+                                          }
+                                Example 2: a simple JSON file for meta device(AUTO/MULTI) with HW device properties.
+                                          {
+                                          	"AUTO": {
+                                          		"PERFORMANCE_HINT": "",
+                                          		"PERF_COUNT": "NO",
+                                          		"DEVICE_PROPERTIES": {
+                                          			"CPU": {
+                                          				"INFERENCE_PRECISION_HINT": "f32",
+                                          				"NUM_STREAMS": "3"
+                                          			},
+                                          			"GPU": {
+                                          				"INFERENCE_PRECISION_HINT": "f32",
+                                          				"NUM_STREAMS": "5"
+                                          			}
+                                          		}
+                                          	}
+                                          }
+
+   Statistics dumping options:
+    -report_type "<type>"     Optional. Enable collecting statistics report. "no_counters" report contains configuration options specified, resulting FPS and latency. "average_counters" report extends "no_counters" report and additionally includes average PM counters values for each layer from the network. "detailed_counters" report extends "average_counters" report and additionally includes per-layer PM counters and latency for each executed infer request.
```

Comment:
You've incorrectly resolved merge conflicts. There are two `-report_type` now in README

Reply:
Updated.